src/terms/operators.jl

```diff
@@ -52,7 +52,11 @@ struct RealSpaceMultiplication <: RealFourierOperator
     kpoint
     potential::AbstractArray
 end
-apply!(Hψ, op::RealSpaceMultiplication, ψ) = Hψ.real .+= op.potential .* ψ.real
+function apply!(Hψ, op::RealSpaceMultiplication, ψ)
```

Comment:
It's really a shame this distroys the nice one line. Aliasing to apply_real_space_multiplication?